src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/scenario/JiraCoreScenarioIT.kt

```diff
@@ -32,111 +15,18 @@ class JiraCoreScenarioIT {
      */
     @Test
     fun shouldRunScenarioWithoutErrors() {
-        val version = System.getenv("JIRA_SOFTWARE_VERSION") ?: "8.0.0"
-        logger.info("Testing Jira $version")
-        val scenario = JiraCoreScenario()
-        val metrics = mutableListOf<ActionMetric>()
-        val actionMeter = ActionMeter.Builder(
-            output = CollectionActionMetricOutput(metrics)
-        ).build()
-
-        val user = User("admin", "admin")
-        val userMemory = object : UserMemory {
-            override fun recall(): User {
-                return user
-            }
-
-            override fun remember(memories: Collection<User>) {
-                throw Exception("not implemented")
-            }
-        }
-        var firstBackupElement = true
-        var secondBackupElement = true
+        val jiraVersion = System.getenv("JIRA_SOFTWARE_VERSION") ?: "8.0.0"
```

Comment:
How about accessing the value from `AbstractJiraCoreScenario`? I'm afraid that having it here and there (where it's actually only used to display a log message) will lead to people making mistakes in the future when e.g. changing a default value.

Reply:
Sure!